### PR TITLE
Adjust TEST_TIMEOUT on hypershift-mgmt-critical jobs to avoid timeouts

### PR DESCRIPTION
**Pull request by fxierh**
https://issues.redhat.com/browse/OCPQE-20069

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml [hunk 1]
```diff
@@ -638,18 +638,18 @@ tests:
...
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
       TEST_ADDITIONAL: Hypershift|Network_Observability
       TEST_FILTERS: ~ChkUpgrade&;~DisconnectedOnly&;~MicroShiftOnly&;HyperShiftMGMT&
       TEST_IMPORTANCE: Critical
-      TEST_TIMEOUT: "35"
+      TEST_TIMEOUT: "90"
     test:
-    - chain: openshift-e2e-test-hypershift-qe-mgmt
+    - chain: openshift-e2e-test-hypershift-qe-mgmt-longduration
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:652]: I think `longduration` are mean to run test cases with `longduration`.
If your test cases do not have that tag, and you just want longer execution time, overwrite `TEST_TIMEOUT`, but not change the test name and test chain.

Comment (fxierh) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml]: @liangxia Okay, updated.

Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml]: Perhaps update the PR title, too ?

Comment (fxierh) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml]: Done.

**Pull request merged**